packages/ui-date-time-input/README.md

```diff
@@ -0,0 +1,54 @@
+---
+category: packages
+---
+
+## ui-date-time-input
+
+[![npm][npm]][npm-url]&nbsp;
+[![MIT License][license-badge]][license]&nbsp;
+[![Code of Conduct][coc-badge]][coc]
+
+A component that can be used to enter date and time.
+
+### Components
+
+The `ui-date-time-input` package contains the following:
+
+- [DateTimeInput](#DateTimeInput)
+
+### Installation
+
+```sh
+yarn add @instructure/ui-date-time-input
+```
+
+### Usage
```

Comment:
The package README shouldn't have any usage info, the other packages don't have either, just links to the components that are in the package, e.g.: https://instructure.design/#ui-date-input